qtfred/src/main.cpp

```diff
@@ -119,38 +113,60 @@ int main(int argc, char* argv[]) {
+	SubsystemMap initializers = {{ SubSystem::OS,                app.tr("Initializing OS interface") },
+								 { SubSystem::CommandLine,       app.tr("Parsing command line") },
+								 { SubSystem::Timer,             app.tr("Initializing Timer") },
+								 { SubSystem::CFile,             app.tr("Initializing CFile") },
+								 { SubSystem::Locale,            app.tr("Initializing locale") },
+								 { SubSystem::Sound,             app.tr("Initializing sound") },
+								 { SubSystem::Graphics,          app.tr("Initializing graphics") },
+								 { SubSystem::Scripting,         app.tr("Initializing scripting") },
+								 { SubSystem::Fonts,             app.tr("Initializing Fonts") },
+								 { SubSystem::Keyboard,          app.tr("Initializing keyboard") },
+								 { SubSystem::Mouse,             app.tr("Initializing mouse") },
+								 { SubSystem::Particles,         app.tr("Initializing particles") },
+								 { SubSystem::Iff,               app.tr("Initializing IFF") },
+								 { SubSystem::Objects,           app.tr("Initializing objects") },
+								 { SubSystem::Models,            app.tr("Initializing model system") },
+								 { SubSystem::Species,           app.tr("Initializing species") },
+								 { SubSystem::BriefingIcons,     app.tr("Initializing briefing icons") },
+								 { SubSystem::HudCommOrders,     app.tr("Initializing HUD comm orders") },
+								 { SubSystem::AlphaColors,       app.tr("Initializing alpha colors") },
+								 { SubSystem::GameSound,         app.tr("Initializing briefing icons") },
+								 { SubSystem::MissionBrief,      app.tr("Initializing briefings") },
+								 { SubSystem::AI,                app.tr("Initializing AI") },
+								 { SubSystem::AIProfiles,        app.tr("Initializing AI profiles") },
+								 { SubSystem::Armor,             app.tr("Initializing armors") },
+								 { SubSystem::Weapon,            app.tr("Initializing weaponry") },
+								 { SubSystem::Medals,            app.tr("Initializing medals") },
+								 { SubSystem::Glowpoints,        app.tr("Initializing glow points") },
+								 { SubSystem::Ships,             app.tr("Initializing ships") },
+								 { SubSystem::Parse,             app.tr("Initializing parser") },
+								 { SubSystem::TechroomIntel,     app.tr("Initializing techroom intel") },
+								 { SubSystem::Nebulas,           app.tr("Initializing nebulas") },
+								 { SubSystem::Stars,             app.tr("Initializing stars") },
+								 { SubSystem::Ssm,               app.tr("Initializing SSMs") },
+								 { SubSystem::EventMusic,        app.tr("Initializing event music") },
+								 { SubSystem::FictionViewer,     app.tr("Initializing fiction viewer") },
+								 { SubSystem::CommandBriefing,   app.tr("Initializing command briefing") },
+								 { SubSystem::Campaign,          app.tr("Initializing campaign system") },
+								 { SubSystem::NebulaLighting,    app.tr("Initializing nebula lighting") },
```

Comment:
Maybe I misread things, but shouldn't this be NebulaLightning?

Reply:
It seems like I misunderstood what that system was for...